### ISSUE TITLE
Username not showing in navbar

### ISSUE THREAD

**Issue by jat255** [openanalytics/shinyproxy#267]
I have an interesting issue on the tap navbar where there is no username next to the user icon. This does not work when using either LDAP or SAML for authentication. This looks like it might be a problem with ThymeLeaf, as this is the code that controls that in the navbar fragment:

```html
<div class="navbar-right" sec:authorize="isAuthenticated()">
	<span class="navbar-text">
		<span class="glyphicon glyphicon-user"></span>
		<span sec:authentication="name"></span>
	</span>
</div>
```

Any ideas why this is happening? I figure I’m just missing a config setting somewhere. (Note, the `SHINYPROXY_USERNAME` variable is being set correctly, so that part appears working using the name-attribute setting, but the Spring framework isn't recognizing the `name` attribute. The user icon is showing, so that `isAuthenticated()` is working properly, but the `sec:authentication` looks like it is not). 

This is with ShinyProxy 2.4.3 and containerproxy 0.8.7

**Comment by LEDfan** [comment]
Hi

Thanks for opening this issue, this is indeed a bug we discovered. It's already fixed in the development version and we plan to release it with the next release. This the commit that fixed the issue: https://github.com/openanalytics/containerproxy/commit/0b79bcc539bf6b5ac7a8a7aaed727d9716538d99

I build a fresh ShinyProxy with only this change for you: https://cloud.openanalytics.eu/s/wQNwaQJpZKmjLRn , of course you can build the JAR yourself too.

**Comment by jat255** [comment]
Great, thanks. For now I'll just wait until the next release. Thank you!

**Comment by LEDfan** [comment]
Hi @jat255 

We released ShinyProxy 2.5.0 in which this bug was fixed.
